model/src/main/proto/feedback_reporting.proto

```diff
@@ -0,0 +1,136 @@
+syntax = "proto3";
+
+package model;
+
+import "profile.proto";
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Top-level proto to store feedback reports locally when there is no network connection to upload
+// them to the remote service
```

Comment:
Nit: end doc comment with punctuation for consistency.

Reply:
Done